### PR TITLE
AAE-9036 add new identity adapter service

### PR DESCRIPTION
**Pull request by jesty**

### CODE REVIEW

File: helm/alfresco-process-infrastructure/values.yaml [hunk 1]
```diff
@@ -969,3 +969,51 @@ opensearch:
...
+    pullPolicy: Always
+  liquibase:
+    enabled: false
+  postgresql:
+    enabled: false
+  probePath: /actuator/health
+  rabbitmq:
+    enabled: false
+  activiti:
+    keycloak:
```
Comment (miguelruizdev) [inline helm/alfresco-process-infrastructure/values.yaml:993]: is there any triaging for the tasks about the removal of Keycloak refences from the infrastructure -like this one- and env vars? @jesty

**Review by miguelruizdev — APPROVED** [review]

File: helm/alfresco-process-infrastructure/values.yaml [hunk 2]
```diff
@@ -969,3 +969,51 @@ opensearch:
...
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: nginx
+      nginx.ingress.kubernetes.io/rewrite-target: /$1
+    path: "/identity-adapter-service/?(.*)"
+  image:
+    repository: quay.io/alfresco/alfresco-identity-adapter-service
+    tag: 7.5.0-alpha.1
+    pullPolicy: Always
```
Comment (erdemedeiros) [inline helm/alfresco-process-infrastructure/values.yaml:984]: Actually, now that we are using alphas, I think we could use `ifNotPresent`, as the versions will always be different. What do you think?

**Review by erdemedeiros — APPROVED** [review]
LGTM

**Review by EquinoxBlack — APPROVED** [review]

**Review by gicappa — APPROVED** [review]
LGTM!

File: helm/alfresco-process-infrastructure/values.yaml (hunk 1 above)
Comment (jesty) [reply helm/alfresco-process-infrastructure/values.yaml:993]: there is a separate issue for it, I prefer to don't misc stuff

File: helm/alfresco-process-infrastructure/values.yaml (hunk 2 above)
Comment (jesty) [reply helm/alfresco-process-infrastructure/values.yaml:984]: I guess we can change everything in another PR, I prefer to don't to do it in a different way then others

**Pull request merged**